cmd/common.go

```diff
@@ -36,35 +37,25 @@ func SetStopCh(stopCh chan struct{}) {
 
 // workspaceExists checks if workspace exists in Kong.
 func workspaceExists(config utils.KongClientConfig) (bool, error) {
-
-	workspace := config.Workspace
-	if workspace == "" {
-		// default workspace always exists
-		return true, nil
-	}
```

Comment:
Please do not remove this optimization. This saves a round-trip to Kong for all Kong OSS installations.

Reply:
942b42b